storybook/src/foundation/assets/font.docs.mdx

```diff
@@ -2,16 +2,16 @@
 
 import { Meta, Typeset } from "@storybook/blocks";
 
-<Meta title="Docs/Assets/Font" />
+<Meta title="Foundation/Assets/Font" />
 
 # Font
 
-We use the Amsterdam Sans font.
+We have our own typeface: Amsterdam Sans.
```

Comment:
I thought this might be a nice way to use both terms, like you suggested having on between parentheses. Or would it be best to consistently use the less correct term?

Reply:
> Or would it be best to consistently use the less correct term?

I think so, because we say 'the Arial font' and 'the font files' later. Having 2 names for the same thing in the same section is a little confusing imo. We could add something like 'font (or more accurately 'typeface')', but maybe that's overkill.